qutebrowser/mainwindow/mainwindow.py

```diff
@@ -508,8 +518,7 @@ def closeEvent(self, e):
         # Process all quit messages that user must confirm
         if quit_texts or 'always' in confirm_quit:
             text = '\n'.join(['Really quit?'] + quit_texts)
```

Comment:
Should probably be a `'<br/>'` now, and `'Really quit?'` could be the title of the prompt.